cypress/e2e/3-testing/login.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('Login functionality', () => {
+  it('User cannot submit login form with invalid credentials', () => {
+    cy.visit('../../index.html');
+    cy.get('#loginForm input[name="email"]').type('invalidemail@example.com');
+    cy.get('#loginForm input[name="password"]').type('invalidpassword');
+    cy.get('#loginForm button[type="submit"]').click();
+    // cy.wait(5000);
+    // cy.contains('Invalid credentials. Please try again.').should('be.visible');
```

Comment:
You should be checking for an alert here https://www.tutorialspoint.com/cypress/cypress_alerts.htm